packages/styles/scss/patterns/blocks/content-block-simple/_content-block-simple.scss

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../../globals/imports';
+
+@mixin content-block-simple {
+  .#{$prefix}--content-block-simple {
+    &__content {
+      padding-bottom: $carbon--spacing-07;
+    }
+
+    .#{$prefix}--content-item {
+      margin-bottom: $carbon--spacing-07;
+
+      &__heading {
+        margin-top: 0;
+      }
+
+      p {
+        margin-bottom: $carbon--spacing-07;
+
+        &:last-of-type {
+          margin-bottom: 0;
```

Comment:
I'm not sure this is necessary? Especially if you use margin up above. then margins can collapse.

Reply:
Same reason as https://github.com/carbon-design-system/ibm-dotcom-library/pull/1429#discussion_r382719090